src/components/Image/RectangleImage/RectangleImage.jsx

```diff
@@ -0,0 +1,10 @@
+import styles from "./RectangleImage.module.scss";
+import IdolImg from "../../../assets/Image_Idol.png";
+
+export default function RectangleImage() {
+  return (
+    <div className={styles.rectangleImage}>
+      <img src={IdolImg} alt="르세라핌 채원 이미지" />
+    </div>
+  );
+}
```

Comment:
지금은 mock 데이터로 IdolImg를 가져오셨지만,
재사용을 고려하여 Img를 prop으로 받아보는 건 어떨까요?

```suggestion
export default function RectangleImage({ src }) {
  return (
    <div className={styles.rectangleImage}>
      <img src={src} alt="img" />
    </div>
```

사용은 이렇게 할 수 있겠네요!
```js
<RectangleImage src={IdolImg} />
```